<!--
Handles complex message generation from the Arduinos.
-->
<launch>
    <node name="arduino_relay" pkg="ros_homebot" type="arduino_relay.py" output="screen" launch-prefix="nice -n -19">
    </node>
</launch>
